cli/src/main/java/fr/pilato/elasticsearch/crawler/fs/cli/FsCrawler.java

```diff
@@ -313,6 +304,23 @@ public static void moveLegacyResources(Path root) {
         }
     }
 
+    private static void checkForDeprecatedResources(Path configDir, Version elasticsearchVersion) throws IOException {
+        try {
+            // If we are able to read an old configuration file, we should tell the user to check the documentation
+            readDefaultJsonVersionedFile(configDir, Byte.toString(elasticsearchVersion.major), "doc");
+            logger.warn("We found old configuration index settings: [{}/_default/doc.json]. You should look at the documentation" +
+                    " about upgrades: https://fscrawler.readthedocs.io/en/latest/installation.html#upgrade-fscrawler.",
+                    configDir);
+        } catch (IllegalArgumentException ignored) { }
+        try {
+            // If we are able to read an old configuration file, we should tell the user to check the documentation
+            readDefaultJsonVersionedFile(configDir, Byte.toString(elasticsearchVersion.major), "folder");
+            logger.warn("We found old configuration index settings: [{}/_default/folder.json]. You should look at the documentation" +
+                    " about upgrades: https://fscrawler.readthedocs.io/en/latest/installation.html#upgrade-fscrawler.",
+                    configDir);
+        } catch (IllegalArgumentException ignored) { }
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Either remove or fill this block of code. ([squid:S00108](https://sonarcloud.io/organizations/dadoonet-github/rules#rule_key=squid%3AS00108))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fr.pilato.elasticsearch.crawler%3Afscrawler-parent&pullRequest=579&issues=AWTql2IgrFWWQMKkjhyH&open=AWTql2IgrFWWQMKkjhyH)

[ISSUE_KEY]: AWTql2IgrFWWQMKkjhyH